src/llmq/blockprocessor.cpp

```diff
@@ -46,7 +46,7 @@ static const std::string DB_MINED_COMMITMENT_BY_INVERSED_HEIGHT_Q_INDEXED = "q_m
 
 static const std::string DB_BEST_BLOCK_UPGRADE = "q_bbu2";
 
-CQuorumBlockProcessor::CQuorumBlockProcessor(CChainState& chainstate, CConnman& _connman, CEvoDB& evoDb, const std::unique_ptr<PeerManager>& peerman) :
+CQuorumBlockProcessor::CQuorumBlockProcessor(CChainState& chainstate, CConnman& _connman, CEvoDB& evoDb, PeerManager& peerman) :
```

Comment:
`peerman` can't be reference here, because this unique_ptr is initialized after `CQuorumBlockProcessor` created.

See https://github.com/dashpay/dash/pull/5782 and https://github.com/dashpay/dash/pull/5792 for related refactorings